src/uct/cuda/base/cuda_iface.h

```diff
@@ -50,6 +50,23 @@
     })
 
 
+#define UCT_CUDADRV_CTX_ACTIVE(_state)                             \
+    do {                                                           \
+        CUcontext cur_ctx;                                         \
+        CUdevice dev;                                              \
+        unsigned int flags;                                        \
+        _state = 0;                                                \
```

Comment:
Pls add empty line after variable declarations and do
``` 
unsigned int flags;

_state = 0;
```

Btw, we often use `unsigned` instead of `unsigned int`